include/Make/GuiScript.make

```diff
@@ -8,8 +8,12 @@ include $(MODULE_TOPDIR)/include/Make/ScriptRules.make
 include $(MODULE_TOPDIR)/include/Make/HtmlRules.make
 
 MODULES  := $(patsubst g.gui.%.py,%,$(wildcard g.gui.*.py))
+### to be removed
 CMDHTML  := $(patsubst %,$(HTMLDIR)/g.gui.%.html,$(MODULES))
 GUIHTML  := $(patsubst %,$(HTMLDIR)/wxGUI.%.html,$(MODULES))
+###
+CMDMAN := $(patsubst %,$(MDDIR)/source/g.gui.%.md,$(MODULES))
+GUIMAN  := $(patsubst %,$(MDDIR)/source/wxGUI.%.md,$(MODULES))
```

Comment:
Maybe `CMDDOC` instead of MAN since we have also the output for a tool called _man_?

Reply:
Same for the Python code in man. Of course, there is the man dir, but let's start somewhere and make the new code more clear.

Reply:
I was also thinking about `CMDDOC`, but there is `doc` directory and `include/Make/Docs.make`...

Reply:
@wenzeslaus Let's keep it for separated PR (remove HTML-related rules & clean-up)